Dockerfile

```
@@ -0,0 +1,39 @@
+
+# Copy project files
+ADD . /app
+
+
+# Sync dependencies
+RUN uv sync --frozen && uv sync --group prod --frozen
+
+# Collect static files
+RUN mkdir -p /app/staticfiles
+
+RUN chmod +x /app/entrypoint.sh
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)